### ISSUE TITLE
Plot disappears when changing Target Variable:

### ISSUE THREAD

**Issue by jarad** [nicebread/corona#33]
Both at http://shinyapps.org/apps/corona/ and using the cloned repository on my local computer, when I change the Target Variable, the plot disappears. Even going back to the original Target Variable, the original plot does not come back.

**Comment by jarad** [comment]
Looks like the "Countries to Display" gets unchecked for all countries.

**Comment by nicebread** [comment]
Yes, this is a bug. If you want to submit a pull request with a fix, go ahead :-)
